testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -413,7 +413,7 @@ public void testReferencedTablesWithMaterializedViews()
     public void testReferencedTablesInCreateView()
             throws Exception
     {
-        runQueryAndWaitForEvents("CREATE VIEW mock.default.test_view AS SELECT * FROM nation", 2);
+        runQueryAndWaitForEvents("CREATE OR REPLACE VIEW mock.default.test_view AS SELECT * FROM nation", 2);
```

Comment:
is this change needed?

Reply:
Because the view named test_view  is add to ``MockConnectorMetadata`` by withGetViews in ``TestEventListenerBasic``, so ``metadata.isViews`` will always return true and throw a exception

Reply:
> Because the view named test_view is add to MockConnectorMetadata by withGetViews in TestEventListenerBasic, so metadata.isViews will always return true and throw a exception

Should use use different view name here then?